readme.md

```diff
@@ -1,4 +1,4 @@
-# Crud Resource for Backpack
+# Fluent Interface for Laravel Backpack
```

Comment:
Personally, I would avoid calling this _the_ "Fluent Interface", even thought... it is. That's because we already have the [Fluent API](https://backpackforlaravel.com/docs/5.x/crud-fluent-syntax) and it will probably be confusing, both for developers, and for ourselves, to know when you're talking about "fluent interface"... which one are you talking about?

How about... 
- Resource Interface? 
- Resource Syntax? 
- True Fluent Syntax/Interface? 
- Advanced Fluent Interface?
- Advanced Fluent Syntax?

Something along those lines should make it easy to use this in parallel with the built-in ones:
- array syntax
- fluent syntax

Personally, I like `resource syntax` and `advanced fluent syntax` the best. I find "interface" is confusing as a word, makes me think of user interface more than developer interface.

Sorry for the rant. Just wanted to help out 😅 You do you.